packages/in_app_purchase/in_app_purchase_storekit/ios/Classes/FIAPaymentQueueHandler.m

```diff
@@ -4,18 +4,49 @@
 
 #import "FIAPaymentQueueHandler.h"
 #import "FIAPPaymentQueueDelegate.h"
+#import "FIATransactionCache.h"
 
 @interface FIAPaymentQueueHandler ()
 
+/// The SKPaymentQueue instance connected to the App Store and responsible for processing
+/// transactions.
 @property(strong, nonatomic) SKPaymentQueue *queue;
+
+/// Callback method that is called each time the App Store indicates transactions are updated.
 @property(nullable, copy, nonatomic) TransactionsUpdated transactionsUpdated;
+
+/// Callback method that is called each time the App Store indicates transactions are removed.
 @property(nullable, copy, nonatomic) TransactionsRemoved transactionsRemoved;
+
+/// Callback method that is called each time the App Store indicates transactions failed to restore.
 @property(nullable, copy, nonatomic) RestoreTransactionFailed restoreTransactionFailed;
+
+/// Callback method that is called each time the App Store indicates restoring of transactions has
+/// finished.
 @property(nullable, copy, nonatomic)
     RestoreCompletedTransactionsFinished paymentQueueRestoreCompletedTransactionsFinished;
+
+/// Callback method that is called each time an in-app purchase has been initiated from the App
+/// Store.
 @property(nullable, copy, nonatomic) ShouldAddStorePayment shouldAddStorePayment;
+
+/// Callback method that is called each time the App Store indicates downloads are updated.
 @property(nullable, copy, nonatomic) UpdatedDownloads updatedDownloads;
 
+/// The transaction cache responsible for caching transactions.
+///
+/// Keeps track of transactions that arrive when the Flutter client is not
+/// actively observing for transactions.
+@property(strong, nonatomic, nonnull) FIATransactionCache *transactionCache;
+
+/// Indicates if the Flutter client is observing transactions.
+///
+/// When the client is not observing, transactions are cached and send to the
+/// client as soon as it starts observing. The Flutter client can start
+/// observing by sending a startObservingPaymentQueue message and stop by
+/// sending a stopObservingPaymentQueue message.
+@property(atomic, assign, readwrite) BOOL isObservingTransactions;
```

Comment:
Properties should not start with `is`, only the getter for the property should start with `is`. See https://google.github.io/styleguide/objcguide.html#objective-c-method-names

Reply:
Fixed.